EIPS/eip-draft-hdwallets.md

```diff
@@ -0,0 +1,76 @@
+## Preamble
+
+    EIP: <to be assigned>
+    Title: Ethereum hierarchy for deterministic wallets
+    Author: Nick Johnson <nick@ethereum.org>, Micah Zoltu
+    Type: Standard Track
+    Category : ERC
+    Status: Draft
+    Created: 2017-04-13
+    Replaces: [84](https://github.com/ethereum/EIPs/issues/84)
+
+
+## Abstract
+This EIP defines a logical hierarchy for deterministic wallets based on [BIP32](https://github.com/bitcoin/bips/blob/master/bip-0032.mediawiki), the purpose scheme defined in [BIP43](https://github.com/bitcoin/bips/blob/master/bip-0043.mediawiki) and eip-draft-ethereum-purpose.
+
+This EIP is a particular application of eip-draft-ethereum-purpose.
+
+## Motivation
+At present, different Ethereum clients and wallets use different derivation paths; a summary of them can be found [here](https://github.com/ethereum/EIPs/issues/84#issuecomment-292324521). Some of these paths violate BIP44, the standard defining derivation paths starting with `m/44'/`. This creates confusion and incompatibility between wallet implementations, in some cases making funds from one wallet inaccessible on another, and in others requiring prompting users manually for a derivation path, which hinders usability.
+
+Further, BIP44 was designed with UTXO-based blockchains in mind, and is a poor fit for Ethereum, which uses an accounts abstraction instead.
+
+As an alternative, we propose a deterministic wallet hierarchy better tailored to Ethereum's unique requiremnts.
+
+## Specification
+We define the following 3 levels in BIP32 path:
+
+<pre>
+m / purpose' / subpurpose' / wallet'
+</pre>
+
+Apostrophe in the path indicates that BIP32 hardened derivation is used.
+
+Each level has a special meaning, described in the chapters below.
+
+### Purpose
+
+Purpose is a constant set to a value to be determined; the authors of this EIP are still engaged in the standards process to obtain assignment of a 'purpose' field.
+
+The purpose field indicates that the subtree of this node is used according to this specification.
+
+Hardened derivation is used at this level.
+
+### Subpurpose
+Subpurpose is set to the EIP number specifying the remainder of the BIP32 derivation path. For paths following this EIP specification, the number assigned to this EIP is used.
+
+### Wallet
+This component of the path splits the wallet into different user identities, allowing a single wallet to have multiple public identities.
+
+Accounts are numbered from index 0 in sequentially increasing manner. This number is used as child index in BIP32 derivation.
+
+Hardened derivation is used at this level.
+
+Software should prevent a creation of an account if a previous account does not have a transaction history (meaning its address has not been used before).
```

Comment:
Or you might want to add a gap concept similar to BIP 44 here https://github.com/bitcoin/bips/blob/master/bip-0044.mediawiki#Address_gap_limit